pom.xml

```diff
@@ -64,6 +64,7 @@
     <module>zookeeper-client</module>
     <module>zookeeper-recipes</module>
     <module>zookeeper-assembly</module>
+      <module>zookeeper-compatibility-tests</module>
```

Comment:
It is good to have this top level module.
I feel we will have more modules of this type in the future.
So I would like to see a separate module for curator

Reply:
So you want a 3rd level module? Something like:

```
root
     zookeeper-compatibility-tests
        zookeeper-compatibility-test-curator
```

What purpose, then, does the parent `zookeeper-compatibility-tests` module serve?

Reply:
It will be container of future compatibility modules like yours.

We won't have only Curator one day.

Doesn't  it make sense to you?

Reply:
I guess it could serve to make sure there are no dependency incompatibilities between components being tested.